app/controllers/questions_controller.rb

```diff
@@ -29,7 +29,7 @@ def new
 
   def create
     permitted_params = params.permit(:category_id, :text, :title, :user_id)
-    permitted_params[:user_id] = User.first.id # change when implement sign_in/sessions
+    permitted_params[:user_id] = current_user['id'] # change when implement sign_in/sessions
```

Comment:
May there be a situation where `current_user` is `nil`?

Reply:
No, if user is not logged in, creation of the answer/question is impossible. But I will add check just in case

Reply:
@o-volosianskyi :+1:

Reply:
OK, do it either in this pull request or within the next :)